erpnext/regional/united_arab_emirates/utils.py

```diff
@@ -7,18 +7,41 @@
 
 
 def update_itemised_tax_data(doc):
-	# maybe this should be a standard function rather than a regional one
-	if not doc.taxes:
-		return
-
 	if not doc.items:
 		return
 
+	has_zero_rated_field = False
 	meta = frappe.get_meta(doc.items[0].doctype)
 	if not meta.has_field("tax_rate"):
 		return
 
+	if meta.has_field("is_zero_rated"):
+		has_zero_rated_field = True
+
 	itemised_tax = get_itemised_tax(doc.taxes)
+	doc.flags.export_determined = None
+	is_export = False
```

Comment:
This too can be removed

Reply:
Have kept that variable to avoid recomputing